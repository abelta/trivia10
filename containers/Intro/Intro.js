import React from 'react';
import { Subtitle, Text, Title, View } from '@shoutem/ui';
import { Link } from 'react-router-native';

const Intro = () => (
  <View styleName="flexible">
    <Title>Welcome to the Trivia Challenge</Title>
    <Subtitle>You will be presented with 10 True or False questions.</Subtitle>
    <Subtitle>Can you score 100%?</Subtitle>
    <Link to="/initialize"><Text>BEGIN</Text></Link>
  </View>
);

export default Intro;
