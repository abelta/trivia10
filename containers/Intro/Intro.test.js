import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter as Router } from 'react-router-native';
import Intro from '.';

describe('Intro', () => {
  const tree = renderer.create(
    <Router>
      <Intro />
    </Router>,
  );

  it('renders properly', () => {
    expect(tree).toMatchSnapshot();
  });
});
