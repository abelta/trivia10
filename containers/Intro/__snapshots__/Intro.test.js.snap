// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Intro renders properly 1`] = `
<View
  animationOptions={Object {}}
  onLayout={[Function]}
  style={
    Object {
      "flex": 1,
    }
  }
  styleName="flexible"
  virtual={undefined}
>
  <View
    animationOptions={Object {}}
    onLayout={[Function]}
    style={
      Object {
        "flex": 1,
      }
    }
    styleName="flexible"
    virtual={undefined}
  >
    <Image
      animationOptions={Object {}}
      onLayout={[Function]}
      source={1}
      style={
        Object {
          "alignItems": "center",
          "flexDirection": "column",
          "height": "100%",
          "justifyContent": "center",
          "resizeMode": "contain",
          "width": "100%",
        }
      }
      virtual={undefined}
    />
  </View>
  <View
    animationOptions={Object {}}
    onLayout={[Function]}
    style={
      Object {
        "flex": 1,
      }
    }
    styleName="flexible"
    virtual={undefined}
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      animationOptions={Object {}}
      disabled={false}
      ellipsizeMode="tail"
      onLayout={[Function]}
      style={
        Object {
          "backgroundColor": "transparent",
          "color": "#222222",
          "fontFamily": "Rubik-Regular",
          "fontSize": 25,
          "fontStyle": "normal",
          "fontWeight": "normal",
          "lineHeight": 30,
        }
      }
      virtual={undefined}
    >
      Welcome to the Trivia Challenge
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      animationOptions={Object {}}
      disabled={false}
      ellipsizeMode="tail"
      onLayout={[Function]}
      style={
        Object {
          "backgroundColor": "transparent",
          "color": "#222222",
          "fontFamily": "Rubik-Regular",
          "fontSize": 20,
          "fontStyle": "normal",
          "fontWeight": "normal",
          "lineHeight": 25,
        }
      }
      virtual={undefined}
    >
      You will be presented with 10 True or False questions.
    </Text>
  </View>
  <View
    animationOptions={Object {}}
    onLayout={[Function]}
    style={
      Object {
        "flex": 1,
      }
    }
    styleName="flexible"
    virtual={undefined}
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      animationOptions={Object {}}
      disabled={false}
      ellipsizeMode="tail"
      onLayout={[Function]}
      style={
        Object {
          "backgroundColor": "transparent",
          "color": "#222222",
          "fontFamily": "Rubik-Regular",
          "fontSize": 20,
          "fontStyle": "normal",
          "fontWeight": "normal",
          "lineHeight": 25,
        }
      }
      virtual={undefined}
    >
      Can you score 100%?
    </Text>
    <View
      animationOptions={Object {}}
      onLayout={[Function]}
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#FFFFFF",
          "flexDirection": "row",
          "flexGrow": 1,
          "flexShrink": 0,
          "justifyContent": "flex-start",
          "paddingBottom": 15,
          "paddingLeft": 15,
          "paddingRight": 15,
          "paddingTop": 15,
        }
      }
      virtual={undefined}
    >
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        collapsable={undefined}
        hitSlop={undefined}
        isTVSelectable={true}
        nativeID={undefined}
        onLayout={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#ffffff",
            "borderColor": "#ffffff",
            "borderRadius": 2,
            "borderWidth": 1,
            "flexDirection": "row",
            "justifyContent": "center",
            "opacity": 1,
            "paddingLeft": 15,
            "paddingRight": 5,
          }
        }
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          animationOptions={Object {}}
          disabled={false}
          ellipsizeMode="tail"
          onLayout={[Function]}
          style={
            Object {
              "backgroundColor": "transparent",
              "color": "#222222",
              "fontFamily": "Rubik-Regular",
              "fontSize": 20,
              "fontStyle": "normal",
              "fontWeight": "normal",
              "lineHeight": 25,
            }
          }
          virtual={undefined}
        >
          BEGIN
        </Text>
      </View>
    </View>
  </View>
</View>
`;
